Documentation/eliminating-pre-builts.md

```diff
@@ -0,0 +1,60 @@
+# Eliminating pre-builts in .NET repositories
+
+This is a detailed guide on how to eliminate pre-builts in a source-buildable repository.
+It is primarily intended for developers contributing to the `dotnet` organization.
+
+## Table of content
+  - [Pre-builts](#pre-builts)
+  - [Elimitating pre-builts](#eliminating-pre-builts)
+  - [FAQ](#faq)
+  - [Contacts](#contacts)
+
+# Pre-builts
+
+_Source-build_ is a process of building a given product on a single machine from source with no internet access.
+
+By definition, _pre-builts_ are dependencies that a repo has on binary files that are not built from source, where _build from source_ points to any package produced during the _current source-build_ or _already present_ in the distro at the time of building (such as `cmake`).
+In layman terms, this means that packages from `nuget.org`, Microsoft builds or other non-source-built binaries cannot be used for source-building a given repository.
+
+When adding a new dependency to a source-buildable repository, the contributor runs the risk of introducing a new pre-built to the product. 
```

Comment:
What do you think about adding a section for how to check for prebuilts?  I mention this because you talk about it here but don't show the command to perform the check locally, that is included in the Eliminate Prebuilts section.

Reply:
It is part of the Eliminating Pre-builts section https://github.com/dotnet/source-build/pull/3083/files#diff-cad8862eb2c4f7cfdba84792f9d81056ffb6724be67b88c2889a83629d038863R71, where we instruct the reader to source-built and check the output. Can expend it even more if needed